src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -89,10 +92,12 @@ private String getToken() {
    * @return a List of IexHistoricalPrices objects for the given symbol,
    *         from the given date until the given date plus range
    */
+  @Cacheable(cacheNames = "historical", sync = true)
   public List<IexHistoricalPrices> getHistoricalPrices(
       final String symbol,
       final String range,
       final String date) {
+    log.info("DID NOT HIT CACHE");
```

Comment:
Nice log. Should probably clean up a bit (not all caps)

Reply:
Sure!